main/src/main/java/cgeo/geocaching/ui/dialog/SimpleDialog.java

```diff
@@ -474,26 +463,83 @@ public final <T> void selectMultiple(@NonNull final List<T> items, @NonNull fina
 
         final AlertDialog dialog = builder.create();
         dialog.show();
-        checkOkButton(dialog, nonEmptyResultRequired, result.size());
+
```

Comment:
The way you added the field it will be:
* displayed in ALL multiple choice dialogs. I would consider making this dynamic (e.g. only show if there are more than 5 entries)
* not be displayable in single-choice dialogs. There might be such dialogs with many choices where such a filter also makes sense (e.g. filter selection in advanced filter dialog)

Reply:
Thanks for reviewing the PR.

> The way you added the field it will be:
> * displayed in ALL multiple choice dialogs. I would consider making this dynamic (e.g. only show if there are more than 5 entries)

Good hint, I'll add a check.

> * not be displayable in single-choice dialogs. There might be such dialogs with many choices where such a filter also makes sense (e.g. filter selection in advanced filter dialog)

Done intentionally for now. I wanted to make it work for multi selections first and get some feedback. Single selection dialogs have some different inner workings, will work on adaption for it next.